src/SmartComponents/AvailableTasks/AvailableTasksTable.js

```diff
@@ -37,46 +34,41 @@ const AvailableTasksTable = ({ availableTasks, error, openTaskModal }) => {
         />
       ) : (
         availableTasks?.map((task) => {
```

Comment:
@johnsonm325 because you are just returning JSX inside the map function, you drop the {} and the return, and just insert the JSX inside () after the arrow